service/dap/server_test.go

```diff
@@ -1654,6 +1655,41 @@ func TestSetBreakpoint(t *testing.T) {
 	})
 }
 
+// TestWorkingDir executes to a breakpoint and tests that the specified
+// working directory is the one used to run the program.
+func TestWorkingDir(t *testing.T) {
+	runTest(t, "workdir", func(client *daptest.Client, fixture protest.Fixture) {
+		fixtureWorkingDir, err := filepath.Abs(filepath.Join(protest.FindFixturesDir(), "buildtest"))
+		if err != nil {
+			t.Fatalf("failed to get working directory: %e", err)
```

Comment:
This message confused me. For a second this check was designed to look up the working directory of the fixture program running under runTest, which doesn't make sense until later in the test. Can we change it to "failed to get fixture directory" please?